client/src/components/HistoryViewer/HistoryViewerVersionDetail.js

```diff
@@ -43,17 +43,20 @@ class HistoryViewerVersionDetail extends PureComponent {
     );
   }
 
+  isPreviewable() {
+    const { isPreviewable, compareMode } = this.props;
+    return isPreviewable && !compareMode;
```

Comment:
The reason I made this change was that
1) Versions are normally previewable, and I want to prevent that while showing a diff form
2) I anticipated issues with it (such as those manifesting), so I abstracted it to make changes easy :P

If isPreviewable shows true, and the preview component renders while in compare mode (which I simply assumed would be the case, to be fair) then which version do we preview? Or do we show a full diff of the preview? If we show a full diff of the preview, then how do we selectively show what is and isn't diff'd? E.g. Elemental content blocks would only be a _portion_ or the preview that requires diffing, not the whole preview (which would be the block page).

If my assumption is entirely wrong, then that's cool, this is mostly irrelevant I suppose.

Reply:
Ok, that's fine if the case is "we shouldn't show previews when in compare mode, even if the object is previewable".

To be fair though, this could also be logic supplied higher up where `isPreviewable` is passed down to this component. I think it originates as a data attribute from entwine, but the HistoryViewer component that passes it down could modify it based on "compare mode" being enabled